entity-framework/core/providers/index.md

```diff
@@ -16,45 +16,45 @@ Entity Framework Core can access many different databases through plug-in librar
-| [Devart.Data.Sugar.EFCore](https://www.nuget.org/packages/Devart.Data.Sugar.EFCore)                                  | SugarCRM                                                    | [DevArt](https://www.devart.com/dotconnect/sugarcrm/)                                           | Paid                                                                            | 6, 7, 8           | [docs](https://docs.devart.com/dotconnect/sugarcrm/GettingStarted.html)                                                                                                 |
-| [Devart.Data.Zoho.EFCore](https://www.nuget.org/packages/Devart.Data.Zoho.EFCore)                                    | Zoho CRM                                                    | [DevArt](https://www.devart.com/dotconnect/zohocrm/)                                            | Paid                                                                            | 6, 7, 8           | [docs](https://docs.devart.com/dotconnect/zohocrm/GettingStarted.html)                                                                                                  |
-| [MASES.EntityFrameworkCore.KNet](https://www.nuget.org/packages/MASES.EntityFrameworkCore.KNet/)                     | Apache Kafka                                                | [MASES Group](https://masesgroup.com)                                                           | Trial, Subscription                                                             | 6, 7, 8           | [docs](https://kefcore.masesgroup.com/)                                                                                                                                 |
-| [InterBase](https://www.nuget.org/packages/InterBaseSql.EntityFrameworkCore.InterBase/)                              | InterBase                                                   | [InterBase](https://interbase.com/)                                                             |                                                                                 | 6                 | [docs](https://docwiki.embarcadero.com/InterBase/2020/en/Entity_Framework)                                                                                              |
+| [Devart.Data.MySql.EFCore](https://www.nuget.org/packages/Devart.Data.MySql.EFCore/)                                 | MySQL 5 onwards                                             | [DevArt](https://www.devart.com/dotconnect/mysql/)                                              | Paid                                                                            | 8                 | [docs](https://docs.devart.com/dotconnect/mysql/GettingStarted.html)                                                                                                    |
```

Comment:
I think I'm seeing the Devart providers supporting EF 9 as well (this may have changed since you originally submitted the PR), can you check?

Reply:
Yes, you approved a PR from them with 9 support, I will rebase